src/module/authentication/store/auth-store.ts

```diff
@@ -311,6 +343,10 @@ export class AuthStore {
 						this.updateErrorField("emailError", "No account matches the email address");
 						parsedError = true;
 						break;
+					case "user not activated":
+						this.updateErrorField("activationError", "You can't log in yet. We previously sent an activation email to you at " + this.email + ". Please follow the instructions in that email to activate your account.");
```

Comment:
I can't see this error message in my testing.  Do you?

Reply:
I did at some point. I can see if I broke it

Reply:
I always get a 500 error with `message: "login failed"` so I don't see any of the parsed errors.